File: app/src/main/java/org/worshipsongs/registry/ITabFragment.java
package org.worshipsongs.registry;

import android.os.Bundle;

import org.worshipsongs.listener.SongContentViewListener;

/**
 * Author : Madasamy
 * Version : 3.x.x
 */

public interface ITabFragment
{
    int defaultSortOrder();

    String getTitle();

    boolean checked();

    void setListenerAndBundle(SongContentViewListener songContentViewListener, Bundle bundle);
}

File: app/src/main/java/org/worshipsongs/registry/ITabFragment.kt
package org.worshipsongs.registry

import android.os.Bundle

import org.worshipsongs.listener.SongContentViewListener

/**
 * @author: Madasamy
 * @since: 3.x
 */

interface ITabFragment
{

    val title: String
    fun defaultSortOrder(): Int

    fun checked(): Boolean

    fun setListenerAndBundle(songContentViewListener: SongContentViewListener?, bundle: Bundle)
}
